static/app/views/alerts/rules/issue/details/ruleDetails.tsx

```diff
@@ -174,6 +187,10 @@ class AlertRuleDetails extends AsyncComponent<Props, State> {
     });
   };
 
+  setIncompatible: SetIncompatibleFunction = () => {
+    this.setState({incompatible: true});
```

Comment:
do we need to use state here? should be able to check this on render since there aren't many re-renders on this page